packages/react/src/stories/ic-radio-group.stories.mdx

```diff
@@ -398,3 +387,38 @@ export const Uncontrolled = () => {
     <Uncontrolled />
   </Story>
 </Canvas>
+
+### Additional field rendering
+
+export const RadioOptionsAdditional = () => {
+  const [showAdditional, setShowAdditional] = useState(false);
+  const changeDisplay = () => {
+    setShowAdditional(!showAdditional);
+  }
+  return (
+    <>
+    <IcRadioGroup label='What is your favourite coffee?' name='radio-group-1'>
+      <IcRadioOption value='Item 1' label='Item 1' />
+      <IcRadioOption value='Item 2' label='Item 2'
```

Comment:
Super minor here - would be nice if the label was "item 2 - has additional" so it's more clear what this story is meant to do